include/sdf/Pbr.hh

```diff
@@ -179,6 +179,21 @@ namespace sdf
     /// \param[in] _map Filename of the emissive map.
     public: void SetEmissiveMap(const std::string &_map);
 
+    /// \brief Get the light map filename. This will be an empty string
+    /// if an light map has not been set.
+    /// \return Filename of the light map, or empty string if a light
+    /// map has not been specified.
+    public: std::string LightMap() const;
```

Comment:
I know this matches the name of the SDF element, but I think it would be more informative if this accessor was called `LightMapFileName`

Reply:
ahh I'm going back and forth on this. I'm trying to keep consistency with the sdf spec, the [msg field in proto](https://github.com/ignitionrobotics/ign-msgs/pull/111/files#diff-5b92bdfa5470042fd4bea61ec281e921d753c888c7eb7550337b40cd6f09cd3cR102), and [ign-rendering API](https://github.com/ignitionrobotics/ign-rendering/pull/182/files#diff-aba7143667dc51e9e4c8270875669cabd51bdc6a8be58768afff529dd01b9655R310). I think there is advantage to keeping the the public APIs consistent. If a more informative function name is preferred here, maybe we can also consider deprecating other functions in this class in favor of `*MapFilename()`?

Reply:
ok, I hadn't thought about the API of other packages. it's fine as is